Examples/AR/CppArExample/Android/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
-<?xml version='1.0' encoding='utf-8'?>
-<manifest package="com.esri.arcgisruntime.qt.cpp.ar.example" xmlns:android="http://schemas.android.com/apk/res/android" android:versionName="1.0" android:versionCode="1" android:installLocation="auto">
+<?xml version="1.0"?>
+<manifest package="com.esri.arcgisruntime.qt.qml.ar.example" xmlns:android="http://schemas.android.com/apk/res/android" android:versionName="-- %%INSERT_VERSION_NAME%% --" android:versionCode="-- %%INSERT_VERSION_CODE%% --" android:installLocation="auto">
```

Comment:
hmm, this change looks unexpected - the package name for c++ app

Reply:
Good catch. I'll fix it.